BrainCloudClient/src/BrainCloud/Unity/Scripts/BrainCloudLoginPF.cs

```diff
@@ -23,13 +23,17 @@ public class BrainCloudLoginPF : MonoBehaviour
         private Vector2 _scrollPosition;
         private string _authStatus = "Welcome to brainCloud";
 
+        public static BrainCloudWrapper BrainCloud;
+        
         void Start()
         {
             ///////////////////////////////////////////////////////////////////
             // brainCloud game configuration
             ///////////////////////////////////////////////////////////////////
-
-            BrainCloudWrapper.Initialize();
+            
+            
+            BrainCloud = gameObject.AddComponent<BrainCloudWrapper>();
```

Comment:
i know paul HATES it when he sees the Capital B .. in brainCloud... 
maybe do something like s_brainCloud ? i know its static and public and i would just usually have it as a cap... just trying to use some of my foresight ;)